onnxscript/ir/_external_data.py

```diff
@@ -51,3 +77,228 @@ def set_base_dir(graph: _core.Graph | _core.GraphView, base_dir: str | os.PathLi
     for tensor in _all_tensors(graph, include_attributes=True):
         if isinstance(tensor, _core.ExternalTensor):
             tensor.base_dir = base_dir
+
+
+def _load_external_data_file(
+    tensors: Sequence[_protocols.TensorProtocol],
+    base_path: str | os.PathLike,
+    relative_path: str | os.PathLike,
+) -> list[_protocols.TensorProtocol]:
+    """
+    Load all external data that is at relative_path into memory for the provided model.
```

Comment:
```suggestion
    """Load all external data that is at relative_path into memory for the provided model.
```

please check and fix docstring styles